services/src/main/java/org/keycloak/services/resources/account/AccountConsole.java

```diff
@@ -161,6 +160,26 @@ public Response getMainPage() throws IOException, FreeMarkerException {
         RequiredActionProviderModel updateEmailActionProvider = realm.getRequiredActionProviderByAlias(UserModel.RequiredAction.UPDATE_EMAIL.name());
         map.put("updateEmailActionEnabled", updateEmailActionProvider != null && updateEmailActionProvider.isEnabled());
 
+        final var devServerUrl = Environment.isDevMode() ? System.getenv(ViteManifest.ACCOUNT_VITE_URL) : null;
+
+        if (devServerUrl != null) {
+            map.put("devServerUrl", devServerUrl);
+        }
+
+        final var manifestFile = theme.getResourceAsStream(ViteManifest.MANIFEST_FILE_PATH);
+
+        if (devServerUrl == null && manifestFile != null) {
+            final var manifest = ViteManifest.parseFromInputStream(manifestFile);
+            final var entryChunk = manifest.getEntryChunk();
+            final var entryStyles = entryChunk.css().orElse(new String[] {});
+            final var entryScript = entryChunk.file();
+            final var entryImports = entryChunk.imports().orElse(new String[] {});
+
+            map.put("entryStyles", entryStyles);
+            map.put("entryScript", entryScript);
+            map.put("entryImports", entryImports);
+        }
```

Comment:
Just a nitpick. We might want to make this generic and move it perhaps to some util class to be used both by Account and Admin Console.

Reply:
Agreed. There is a lot of logic here that should be re-used between the themes, not just this code here. In the future I'd like to have something like a `BaseTheme` that encapsulates all the common logic between themes.